proto/src/determined/container/v1/container.proto

```diff
@@ -38,4 +38,10 @@ message Container {
   State state = 3;
   // A list of devices that is being used by this container.
   repeated determined.device.v1.Device devices = 4;
+  // EE user has no RBAC viewing permissions.
+  // If true, we obfuscate:
+  // (1) parent
+  // (2) id
+  // (4) devices
+  bool permissions_disabled = 5;
```

Comment:
question: is this still true? Or this entry omitted?

Reply:
We use this entry in `get_task_name`! Ref [here](https://github.com/determined-ai/determined/pull/8416/commits/facb87dae441182ef40b02929ee37bdc84fae9f4#r1397600886)